### PR TITLE
feat(DomSchema): add <slot> element

### PR DESCRIPTION
**Pull request by DzmitryShylovich**
**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the new behavior?**
Adds support for `<slot>` element (replacement of the deprecated `<content>`) http://w3c.github.io/webcomponents/spec/shadow/#the-slot-element

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by vicb** [comment]
CI fails

File: modules/@angular/compiler/test/schema/schema_extractor.ts [hunk 1]
```diff
@@ -23,7 +23,9 @@ const ALL_HTML_TAGS =
     // https://www.w3.org/TR/html5/index.html
     'a,abbr,address,area,article,aside,audio,b,base,bdi,bdo,blockquote,body,br,button,canvas,caption,cite,code,col,colgroup,data,datalist,dd,del,dfn,div,dl,dt,em,embed,fieldset,figcaption,figure,footer,form,h1,h2,h3,h4,h5,h6,head,header,hr,html,i,iframe,img,input,ins,kbd,keygen,label,legend,li,link,main,map,mark,meta,meter,nav,noscript,object,ol,optgroup,option,output,p,param,pre,progress,q,rb,rp,rt,rtc,ruby,s,samp,script,section,select,small,source,span,strong,style,sub,sup,table,tbody,td,template,textarea,tfoot,th,thead,time,title,tr,track,u,ul,var,video,wbr,' +
     // https://html.spec.whatwg.org/
-    'details,summary,menu,menuitem';
+    'details,summary,menu,menuitem,' +
+    // http://w3c.github.io/webcomponents/spec/shadow/#the-slot-element
+    'slot';
```
Comment (vicb) [inline modules/@angular/compiler/test/schema/schema_extractor.ts]: should it be in `MISSING_FROM_CHROME` below ?

Comment (laco0416) [inline modules/@angular/compiler/test/schema/schema_extractor.ts]: It's already shipped in Chrome v54 (stable).
image

Comment (vicb) [inline modules/@angular/compiler/test/schema/schema_extractor.ts]: Then we should test the Chrome version and disable the failing test for older Chrome versions

**Pull request closed without merging**